client/components/tinymce/index.jsx

```diff
@@ -178,6 +181,12 @@ module.exports = React.createClass( {
 
 		this.localize();
 
+		let toolbar1 = 'wpcom_add_media,';
+		if ( config.isEnabled( 'post-editor/contact-from' ) ) {
+			toolbar1 += 'wpcom_add_contact_form,';
```

Comment:
Not sure whether we'll expand this to include more feature-flagged items, but in the case that we do, it might be more maintainable if we were to construct it as an array and splice the feature-flagged items:

```js
let toolbar1 = [ 'wpcom_add_media', 'formatselect', 'italic' /* ... */ ];
if ( config.isEnabled( 'post-editor/contact-form' ) ) {
	toolbar1.splice( 1, 0, 'wpcom_add_contact_form' );
}

tinymce.init( {
	toolbar1: toolbar1.join()
} );
```